packages/react-conformance/src/types.ts

```diff
@@ -94,6 +94,12 @@ export interface IsConformantOptions<TProps = {}> {
    * This is 'root' by default, and only needs to be specified if it's a slot other than 'root'.
    */
   primarySlot?: keyof TProps | 'root';
+
+  /**
+   * Test will load the first tsconfig.json file working upwards from `tsconfigDir`.
+   * @defaultvalue the directory of the component being tested
```

Comment:
```suggestion
   * @defaultvalue `process.cwd()`
```

Reply:
We decided to change the default from process.cwd() to the individual component directory (the change in createTsProgram.ts).
It makes more sense to look for the tsconfig closer to the component, instead of at the project root.

Reply:
Oh I missed that, sorry